### ISSUE TITLE
classfication loss function

### ISSUE THREAD

**Issue by eragonruan** [argman/EAST#4]
Thanks for sharing this excellent repo. I noticed that the classification loss function you used in code is different from the paper. you use dice coefficient instead of cross entropy. Could you provide more detail on this part?

**Comment by argman** [comment]
I find dice loss is faster to converge and can reduce false positive predictions compared to weighted-logloss
